### PR TITLE
Storage configuration improvements

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: scanner/src/main/kotlin/storages/PostgresStorage.kt [hunk 1]
```diff
@@ -41,7 +41,7 @@ class PostgresStorage(private val connection: Connection, private val schema: St
     /**
      * Initialize the database.
      */
-    fun init() {
+    init {
```
Comment (mnonnenmacher) [inline scanner/src/main/kotlin/storages/PostgresStorage.kt]: I don't like calling the database in the constructor code. While in our case it does not make a big difference, I think in general calling external resources in a constructor is bad practice.

Comment (sschuberth) [reply scanner/src/main/kotlin/storages/PostgresStorage.kt]: How about renaming this function then? From its name I almost thought you *wanted* this to be an `init` block, but accidentally made it a function. I cannot think of a really good name right now which expresses that external resources are involved, but maybe just `initTable` would do?

Comment (mnonnenmacher) [reply scanner/src/main/kotlin/storages/PostgresStorage.kt]: What about `initializeDatabase` or `setupDatabase`?

Comment (sschuberth) [reply scanner/src/main/kotlin/storages/PostgresStorage.kt]: I went for `setupDatabase` as it's shorter 😀

**Pull request merged**